### DISCUSSION TITLE
Considering updating the font to Inter 4.0

### DISCUSSION THREAD

**Discussion by Linerly** [anyproto/contributors#117] [Anytype apps]
I found out that [Inter](https://rsms.me/inter/) (the font that Anytype uses) now has a [true italic variant](https://github.com/rsms/inter/releases/tag/v4.0) — it would be nice to use the new italic variant as well in the app

Here's before:
![Screenshot_20231209_213212](https://github.com/anyproto/contributors/assets/47475676/085236c9-7eab-4d42-b7a2-c2f181287a6f)

…and after:
![Screenshot_20231209_213121](https://github.com/anyproto/contributors/assets/47475676/aaed1136-af54-44b6-a432-b90daec100cf)

**Comment by ra3orblade** [comment]
Updated, thanks
